webpack.config.js

```diff
@@ -114,6 +114,13 @@ module.exports = {
           // use style-loader in development
           fallback: 'style-loader'
         })
+      },
+      {
+        test: /\.(jpe?g|gif|png|svg)$/i,
+        loaders: [
+          'file-loader?hash=sha512&digest=hex&name=/static/images/[name]-[hash].[ext]',
```

Comment:
Haven't used image-webpack-loader before. But looking [at the docs](https://github.com/tcoopman/image-webpack-loader#usage),  I think it would be cleaner to have this expanded out into a JS object like the other module rules in our config so far. Something like:
```javascript
{
  test: /\.(jpe?g|gif|png|svg)$/i,
  use: [
    {
      loader: 'file-loader',
      options: {
        hash: 'sha512',
        digest: 'hex',
        name: 'static/images/[name]-[hash].[ext]'
      }
    },
    {
      loader: 'image-webpack-loader',
      options: {
        bypassOnDebug: true,
      },
    },
  ],
}
```